manual/src/refman/exten.etex

```diff
@@ -131,4 +131,8 @@ that are implemented in OCaml, but not described in chapter \ref{c:refman}.
 %HEVEA\cutname{labeledtuples.html}
 \input{labeledtuples.tex}
 
+\section{s:labeled-tuples}{External types}
```

Comment:
This should probably be a fresh (label?) name:
```suggestion
\section{s:external-types}{External types}
```

Reply:
Thanks! I will absolutely deny that any copy-paste could have taken place.